expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -219,7 +231,7 @@ private static List<Class<?>> listPublicationInstanceTypes() {
 
     @SuppressWarnings("unchecked")
     private static Class<? extends TicketEntry> someOtherTicketType(Class<? extends TicketEntry> ticketType) {
-        return (Class<? extends TicketEntry>) ticketTypeProvider().filter(type -> !ticketType.equals(type))
+        return (Class<? extends TicketEntry>) ticketTypeProvider().filter(type -> !ticketType.equals(type) && !type.equals(DoiRequest.class))
```

Comment:
Method name does not reflect what the method do. For some reason you are avoiding DoiRequest. Maybe name method someOtherTicketTypeBesidesDoiRequest() ?